package.json

```diff
@@ -39,7 +40,8 @@
   "homepage": "https://github.com/fh1ch/node-bacstack#readme",
   "dependencies": {
     "debug": "^3.1.0",
-    "iconv-lite": "^0.4.19"
+    "iconv-lite": "^0.4.19",
+    "repolinter": "bufferoverflow/repolinter#fix/reuse-software"
```

Comment:
We shouldn't merge something which relies on NPM packages on GitHub/special branches.

Reply:
bingo, that's why this is WIP